dashboard/lib/service/appengine_cocoon.dart

```diff
@@ -60,7 +60,7 @@ class AppEngineCocoonService implements CocoonService {
     try {
       final Map<String, dynamic> jsonResponse = jsonDecode(response.body);
       return CocoonResponse<List<CommitStatus>>.data(
-        await compute<List<dynamic>, List<CommitStatus>>(_commitStatusesFromJson, jsonResponse['Statuses']),
+        _commitStatusesFromJson(jsonResponse['Statuses']),
```

Comment:
Why was the `compute` call added originally?  Was this hanging the UI thread?  Is that no longer a concern?  Then again, there are no isolates on the web, which has been our primary delivery platform, so maybe this is obsolete?